services/dashboard/views/apps/app/appdag.js

```diff
@@ -14,19 +14,23 @@ angular.module('dashboard.apps.appmaster')
     var detectPeriod = 15;
     var previousDetectPoint = {appClock: $scope.app.clock, local: moment()};
     var appClockRate = 0;
+    var appClockInited = false;
     $scope.$watch('app.clock', function(nowAppClock) {
       var nowLocal = moment();
       if (nowLocal - previousDetectPoint.local > detectPeriod * 1000) {
-        $interval.cancel(updateClockPromise);
-        if (nowAppClock - previousDetectPoint.appClock > 0) {
-          var clockDelta = nowAppClock - $scope.displayClock;
-          appClockRate = 1000 * (nowLocal - previousDetectPoint.local) /
-            (nowAppClock  - previousDetectPoint.appClock + clockDelta);
-          updateClockPromise = $interval(function () {
-            $scope.displayClock += 1000;
-          }, appClockRate);
+        if (appClockInited) {
```

Comment:
during the interval of two change of app.clock, the display clock should use the simulated clock.